src/staging/validation.py

```diff
@@ -285,3 +285,62 @@ def combine_schemas_validate_full_df(
         else:
             survey_df[column] = survey_df[column].astype(dtypes[column])
         validation_logger.debug(f"{column} after: {survey_df[column].dtype}")
+
+@exception_wrap
+def validate_ultfoc_csv(df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
Instead of creating a new function in the validation.py, I suggest following the method used for reading in the manual_outliers file which involves creating a .toml schema (with datatypes) and using validate_data_with_schema to check the expected columns and enforce the data types. This will also give the required logging messages.